CreeDictionary/API/models.py

```diff
@@ -147,13 +151,29 @@ class SearchResult:
 
     definitions: Tuple["Definition", ...]
 
-    @property
-    def is_inflection(self) -> bool:
+    def serialize(self) -> Dict[str, Any]:
```

Comment:
@eddieantonio This function introduces a lot of hidden attributes and structures in the return value. I have some ideas.

We should write an extra file `schema.py` with a [TypedDict](https://www.python.org/dev/peps/pep-0589/) that describes the serialized result. 
we do have access to TypedDict (try `from typing_extensions import TypedDict` :wink:)

for the time being, I prefer the first because it has higher readibility and IDE syntactical support.

The other proposal is to go full blown json, and write json schema object in with [python-jsonschema](https://github.com/Julian/jsonschema).

Reply:
I'd prefer the first option for now ;)